Demo/src/main/java/com/paypal/android/ui/paypal/PayPalFragment.kt

```diff
@@ -110,8 +109,8 @@ class PayPalFragment : Fragment(), PayPalWebCheckoutListener {
             try {
                 binding.statusText.setText(R.string.creating_order)
 
-                val orderJson = JsonParser.parseString(OrderUtils.orderWithShipping) as JsonObject
-                val order = sdkSampleServerApi.createOrder(orderJson)
+                val orderRequest = OrderUtils.createOrderBuilder("100.0")
```

Comment:
nit - can we make this amount like ~$5 or less? Just on the off chance that someone ever completes a transaction in prod accidentally it'll make sure it's a much smaller amount.